src/docs/FgTableDocs.vue

```diff
@@ -1,39 +1,39 @@
 <template>
   <div>
-    <divider/>
+    <fg-divider/>
 
-    <header-large>
-      Table
-    </header-large>
+    <fg-header-large>
+      FgTable
```

Comment:
I would make this header match the related label used in the side menu and therefore would drop the prefix:
```suggestion
      Table
```